docs/example-scenario/infrastructure/image-modeling.md

```diff
@@ -0,0 +1,115 @@
+
+This architecture also includes Active Directory domain controllers to control access to Azure resources and internal name resolution through the Domain Name System (DNS). Jump boxes provide administrator access to the Windows and Linux VMs that run the solution.
+
+![architecture diagram](./media/architecture-image-modeling.png)
+
+1. User submits a number of images to PhotoScan.
+2. The PhotoScan Scheduler runs on a Windows VM that serves as the head node and directs processing of the user's images.
+3. PhotoScan searches for common points on the photographs and constructs the geometry (mesh) using the PhotoScan processing nodes running on VMs with graphics processing units (GPUs).
+4. Avere vFXT provides a high-performance storage solution on Azure based on Network File System version 3 (NFSv3) and comprised of at least four VMs.
+5. PhotoScan renders the model.
+
+### Components
+
+- [Agisoft PhotoScan](http://www.agisoft.com/): The PhotoScan Scheduler runs on a Windows 2016 Server VM, and the processing nodes use five VMs with GPUs that run CentOS Linux 7.5.
+- [Avere vFXT](/azure/avere-vfxt/avere-vfxt-overview) is a file caching solution that uses object storage and traditional network-attached storage (NAS) to optimize storage of large datasets. It includes:
+  - Avere Controller. This VM executes the script that installs the Avere vFXT cluster and runs Ubuntu 18.04 LTS. The VM can be used later to add or remove cluster nodes and to destroy the cluster as well.
+  - vFXT cluster. At least three VMs are used, one for each of the Avere vFXT nodes based on Avere OS 5.0.2.1. These VMs form the vFXT cluster, which is attached to Azure Blob storage.
+- [Active Directory domain controllers](/azure/active-directory-domain-services/active-directory-ds-overview) allow the host access to domain resources and provide DNS name resolution. Avere vFXT adds _x_ number of A records—for example, each A record in a vFXT cluster points to the IP address of each Avere vFXT node. In this setup, all VMs use the round-robin pattern to access vFXT exports.
+- [Other VMs](/azure/virtual-machines/) serve as jump boxes used by the administrator to access the scheduler and processing nodes. The Windows jumpbox is mandatory to allow the administrator to access the head node via remote desktop protocol. The second jumpbox is optional and runs Linux for administration of the worker nodes.
+- [Network security groups](/azure/virtual-network/manage-network-security-group) (NSGs) on the Jumpbox subnet allow ports 3389 and 22 access to the VMs attached to the Jumpbox subnet.
```

Comment:
May want to mention that the NSG allows access to the PIP only.

Reply:
Done.